src/main/java/com/hazelcast/commandline/member/ProcessExecutor.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.commandline.member;
+
+import com.hazelcast.core.HazelcastException;
+
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.util.List;
+
+/**
+ * Wrapper for {@link ProcessBuilder} for test purposes
+ */
```

Comment:
Is this still for test purposes only? Looks like it's used in production code.

Reply:
Yes, let me change that.